pyaedt/modules/solutions.py

```diff
@@ -2027,111 +1827,104 @@ def polar_plot_3d(
+        else:
+            return x, y, z
 
+    # fmt: off
     @pyaedt_function_handler()
     def polar_plot_3d_pyvista(
-        self,
-        qty_str="RealizedGain",
-        convert_to_db=True,
-        position=None,
-        rotation=None,
-        export_image_path=None,
-        show=True,
-        show_as_standalone=False,
+            self,
+            farfield_quantity="RealizedGain",
+            format_quantity="dB10",
+            rotation=None,
+            export_image_path=None,
+            show=True,
+            show_as_standalone=False,
+            pyvista_object=None,
+            background=None,
+            scale_farfield=None,
+            show_beam_slider=True,
+            show_geometry=True,
+            **kwargs
     ):
+        # fmt: on
         """Create a 3d Polar Plot of Geometry with Radiation Pattern in Pyvista.
 
         Parameters
         ----------
-        qty_str : str, optional
+        farfield_quantity : str, optional
             Quantity to plot. Default `"RealizedGain"`.
-        convert_to_db : bool, optional
-            Either if the quantity has to be converted in db or not. Default is `True`.
+            Available functions are: `"RealizedGain"`, `"RealizedGain_Theta"`, "`RealizedGain_Phi"`, `"rETotal"`,
+            `"rETheta"`, `"rEPhi"`.
+        format_quantity : str, optional
+            Conversion data function.
+            Available functions are: `"dB10"`, `"dB20"`, "`abs"`, `"real"`, `"imag"`, `"norm"`, `"ang"`, `"and_deg"`.
         export_image_path : str, optional
             Full path to image file. Default is None to not export.
-        position : list, optional
-            It can be a list of numpy list of origin of plot. Default is [0,0,0].
         rotation : list, optional
             It can be a list of numpy list of origin of plot.
             Default is [[1., 0., 0.], [0., 1., 0.], [0., 0., 1.]].
         show : bool, optional
             Either if the plot has to be shown or not. Default is `True`.
         show_as_standalone : bool, optional
             Either if the plot has to be shown as standalone or not. Default is `True`.
+        pyvista_object : :class:`Pyvista.Plotter`, optional
+            Pyvista instance defined externally.
+        background : list or str, optional
+            Background color if a list passed or background picture if string passed.
+        scale_farfield : list, optional
+            List with minimum and maximum value of the scale slider.
+        show_beam_slider : bool, optional
+            Whether the theta and phi scan slider is active or not.
+        show_geometry :
+            Whether the show geometry check box is active or not.
 
         Returns
         -------
         bool or :class:`Pyvista.Plotter`
             Return :class:`Pyvista.Plotter` in case show and export_image_path is `False`.
             In other cases return ``True`` when successful.
```

Comment:
```suggestion
            ``True`` when successful. The :class:`Pyvista.Plotter` is returned when ``show`` and
            ``export_image_path`` are ``False``.
```
Do both ``show`` and ``export_image_path`` have to be ``False``? If so, change to "... when both ``show`` and ....." Otherwise, change to when ``show`` or ...."